### PR TITLE
Parametrize load dump asserts

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

File: tests/test_make.py [hunk 1]
```diff
@@ -38,6 +38,45 @@ def dataclass_param(request):
...
+def _assert_dump_load(schema, loaded, dumped):
+    assert schema.loads(schema.dumps(loaded)) == loaded
+
+
+def _assert_load_dump(schema, loaded, dumped):
+    assert schema.dump(schema.load(dumped)) == dumped
+
+
+def fixture_from_dict(name, id_to_value):
+    @pytest.fixture(
```
Comment (python-desert) [inline tests/test_make.py:94]: Could this function get a docstring? If you feel like it maybe some type hints `name: str, id_to_value: t.Dict[str, something]`

Comment (altendky) [reply tests/test_make.py:94]: Yeah, you will presumably have to bug me about type hints regularly. And documentation...

Comment (altendky) [reply tests/test_make.py:94]: I have no idea how far you want to go with typing so I just went, well, pretty far in on it.  Don't hesitate to guide.

Comment (python-desert) [reply tests/test_make.py:94]: Hah! That's more than I was gonna do, but can't hurt I guess?

**Review by ghost — APPROVED** [review]

**Pull request merged**